core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1380,11 +1412,77 @@ class KafkaController(val config: KafkaConfig,
     partitionStateMachine.triggerOnlinePartitionStateChange(topic)
   }
 
+  private def processTopicMinInSyncReplicasConfigChange(topic: String, minInSyncReplicas: Int): Unit = {
```

Comment:
Can you please also consolidate #207 into this PR?
Also, I think it might be good to add me and Efe as the co-authors besides Kyle. Thanks!

Reply:
Added #207 too.
Will add the co-authors when committing.